tests/docker/docker-compose.yml

```diff
@@ -10,12 +10,28 @@ services:
             start_period: 20s
             interval: 10s
             timeout: 10s
-            retries: 20 
+            retries: 40 
         deploy:
             resources:
                 limits:
                     cpus: '1'
                     memory: 8G 
+        # Uncomment the following two lines beginning with the "volumes:" line
+        # and change the directory to test against locally
+        # built packages.
+        #volumes:
+        #- /local/path/to/ub22/packages:/irods_packages
+
+    globus-ubuntu24:
+        build:
+            context: .
+            dockerfile: globus.ubuntu24.Dockerfile
+        depends_on:
+            irods:
+                condition: service_healthy
+        volumes:
+            - ../../:/irods_client_globus_connector 
```

Comment:
If this volume mount is considered read-only, you can enforce that by appending `:ro` to the line.

Consider whether each volume mount needs that.

Reply:
I made this change but noticed that I am putting test files in the irods_client_globus_connector volume mount.

That is not ideal so I will try to change that.

Reply:
I made these are read-only.  In addition I changed the scripts so that temporary test files are created in /tmp.

Reply:
Excellent.